SOURCE/Main_DOODZ.c

```diff
@@ -787,8 +787,8 @@ int main( int nargs, char *args[] ) {
                 }
                 
                 printf("**********************************************\n");
-                if ( model.Newton == 0 ) { printf("*** Picard it. %02d of %02d (step = %05d) ***\n", Nmodel.nit, Nmodel.nit_max, model.step); TrackNewtonSteps[Nmodel.nit] = 0;}
-                if ( model.Newton == 1 ) { printf("*** Newton it. %02d of %02d (step = %05d) ***\n", Nmodel.nit, Nmodel.nit_max, model.step); TrackNewtonSteps[Nmodel.nit] = 1;}
+                if ( model.Newton == 0 ) { printf("*** Picard it. %02d of %02d (step = %05d) ***\n", Nmodel.nit, Nmodel.nit_max, model.step); LogIsNewtonStep[Nmodel.nit] = 0;}
```

Comment:
I've only now realised that there are 2 actions in a single line of code. And a condition
It is very easy to overlook something when it is written as a single line

I'm quite certain that we need {} here

Reply:
I've modified this part. There were already {} but I've now spread the actions on several lines.
Indeed, it's better not to have several actions on the same line.